Gems/Atom/RPI/Code/Tests/Material/MaterialTypeAssetTests.cpp

```diff
@@ -277,7 +307,7 @@ namespace UnitTest
         CheckPropertyValue<uint32_t>(materialTypeAsset, Name{"MyEnum"}, 0u);
     }
 
-    TEST_F(MaterialTypeAssetTests, SetPropertyValues)
+    TEST_F(MaterialTypeAssetTests, ApplySetValues)
```

Comment:
It looks like you accidentally deleted the original "SetPropertyValues" unit test.

Reply:
Good catch, that was indeed by accident! I have restored it, went over all tests again to see if nothing else was out of the ordinary and found that MaterialTypeAssetTests.SetPropertyValues had suffered a similar fate (now restored) and that a duplicate test had snuck in (now removed). I have reorganised the order of the tests such that related tests are close together to improve clarity and hopefully prevent such mishaps in the future.